requirements/system/systemmodel/system-model.tex

```diff
@@ -13,9 +13,11 @@ \subsection{System Model}
   \centering
   \includegraphics[width=\linewidth]{./system/systemmodel/images/accessjob.png}
```

Comment:
Add that the user transmits the job's name to the central server via the SSH connection.
Change the SSH connection to the worker client to an arrow pointing at the worker client and label the arrow "Forward SSH connection"